server/chat-plugins/wifi.tsx

```diff
@@ -141,21 +141,32 @@ class Giveaway extends Rooms.RoomGame {
 		super.destroy();
 	}
 
-	generateReminder() {}
+	generateReminder(joined?: boolean): string | Chat.VNode;
+	generateReminder() {
+		return '';
+	}
 
 	getStyle() {
-		if (this.game === 'BDSP') return `style="background-color: #aa66a9;color: #fff;padding: 2px 4px;"`;
-		return `class="broadcast-blue"`;
+		const css: {[k: string]: string | {[k: string]: string}} = {className: "broadcast-blue"};
+		if (this.game === 'BDSP') css.style = {backgroundColor: '#aa88a9'};
```

Comment:
Always set bg and fg color simultaneously.

Reply:
What do you mean?

Reply:
```suggestion
		if (this.game === 'BDSP') css.style = {background: '#aa88a9', color: '#fff'};
```

In general, when writing CSS, it's useful to set foreground color at the same time as background color. While in this case, it's unlikely to matter, it often works around CSS mistakes where you end up with white-on-white or black-on-black because you messed up inheritance somewhere.

Reply:
That makes sense, though they use the same foreground hex, so does it matter here?

Reply:
Yes, that's why I'm saying, in this case it's unlikely to be a problem. But you can imagine, for instance, server custom CSS which makes `broadcast-blue` a dark-on-light color scheme. Then this would be unreadable if you overrode `background` but left `color` unchanged.

Reply:
(I also prefer `background` over `backgroundColor`. It makes it clearer that you want the entire background to be that color, rather than also having a background image, and it's also shorter.)